charts/schema-registry/templates/deployment.yaml

```diff
@@ -46,9 +46,9 @@ spec:
                 - sh
                 - -c
                 - netstat -ltn 2> /dev/null | grep {{ .Values.service.port }} || ss -ltn 2> /dev/null | grep {{ .Values.service.port }}
-          {{- if or .Values.authentication.enabled .Values.tls.enabled (and .Values.container .Values.container.volume) }}
+          {{- if or (and .Values.authentication.enabled (not .Values.authentication.disableBasicAuthentication)) .Values.tls.enabled (and .Values.container .Values.container.volume) }}
```

Comment:
we should not worry about disable basic authentication in helm charts. we should simply disable auth handler for "basic" method in auth handler manager.

Reply:
The reason for adding this check was to ensure that the userPasswordFile and passwordAuthSecret are not required when the user has disabled basic authentication.